routes/api-routes.js

```diff
@@ -0,0 +1,134 @@
+// Requiring our models and passport as we've configured it
+const db = require("../models");
+const passport = require("../config/passport");
+
+module.exports = function(app) {
```

Comment:
you should organize habits and login info in to separate files or keep them all in the same file... they are spread out kind of awkwardly